test/create_job.test.js

```diff
@@ -0,0 +1,75 @@
+import { assert } from 'chai';
+import { run } from 'syncano-test';
+import dotenv from 'dotenv';
```

Comment:
Should be possible to do `import dotenv/config from 'dotenv';` and not run dontenv.config()